android/src/main/java/com/reactnative/googlefit/HeartrateHistory.java

```diff
@@ -94,29 +100,19 @@ else if (dataReadResult.getDataSets().size() > 0) {
         return map;
     }
 
-    public boolean save(ReadableMap sample) {
-        // TODO: how to save blood pressure?
-
+    public boolean saveBloodGlucose(ReadableMap sample) {
         this.Dataset = createDataForRequest(
-                this.dataType,    // for heart rate, it would be DataType.TYPE_HEART_RATE_BPM
+                this.dataType,
                 DataSource.TYPE_RAW,
-                sample.getDouble("value"),                  // heart rate in bmp
-                (long)sample.getDouble("date"),              // start time
-                (long)sample.getDouble("date"),                // end time
-                TimeUnit.MILLISECONDS                // Time Unit, for example, TimeUnit.MILLISECONDS
+                sample.getDouble("value"),
+                (long)sample.getDouble("date"),
+                TimeUnit.MILLISECONDS
         );
         new InsertAndVerifyDataTask(this.Dataset).execute();
 
         return true;
     }
 
-    public boolean delete(ReadableMap sample) {
```

Comment:
I know it's probably doing nothing right now, but  it's there any reasons you remove it?

Reply:
Sorry, just did it out of habit, I don't like to keep not used code because it takes time and attention (I prefer keep only useful code)
I can return this method if you want

Reply:
> Sorry, just did it out of habit, I don't like to keep not used code because it takes time and attention (I prefer to keep only useful code)
> I can get back this method if you want, just let me know @aboveyunhai

Just bring it back,  because one thing I did recently is to migrate the deprecated code, these old code sometimes can give me some hints about the past and why they exist, before I remove them.